tests/unit/services/modal-test.js

```diff
@@ -99,4 +99,27 @@ module('Unit | Service | modal', (hooks) => {
 		assert.equal(service.get('content').findBy('name', 'foo').get('promise._state'), PENDING);
 		assert.equal(service.get('content').findBy('name', 'bar').get('promise._state'), REJECTED);
 	});
+
+	test('it triggers events when a modal is open/closed', (assert) => {
+		const done = assert.async();
+
+		service.one('open', (modal) => {
+			assert.ok(1, 'modal is open');
+			assert.equal(modal.name, 'foo', 'modal exists as first argument');
+		});
+
+		run(() => {
+			service.open('foo');
+		});
+
+		service.one('close', (key) => {
+			assert.ok(1, 'modal is closed');
```

Comment:
I think I would test this a bit different by making the event handler function a sinon spy and just checking `spy.calledWithArguments('foo')`